src/js/tracks/text-track-settings.js

```diff
@@ -282,6 +288,13 @@ function captionOptionsMenuTemplate() {
             <option value="small-caps">Small Caps</option>
           </select>
         </div> <!-- vjs-font-family -->
+        <div class="vjs-descriptions-playback vjs-tracksetting">
+          <label class="vjs-label">Descriptions</label>
+          <select>
+            <option value="">Visible</option>
+            <option value="screenReaderOnly">Hidden (for screen reader users)</option>
```

Comment:
The default is Visible, set in the [default button handler](https://github.com/ICTASolutions/video.js/blob/descriptions-track-support-for-v5/src/js/tracks/text-track-settings.js#L34). Its value is "", following the pattern of the [Foreground Color selector](https://github.com/ICTASolutions/video.js/blob/descriptions-track-support-for-v5/src/js/tracks/text-track-settings.js#L188). The settings control could be a simple checkbox, but this allows for future expansion to other modes of display. Defaulting to Visible makes the most sense, since most users (those without a screen reader) will see the descriptions if they turn them on, and users with a screen reader in general won't be distracted by visible text.